extensions/github/src/components/RepositoryActions.tsx

```diff
@@ -107,6 +107,13 @@ export default function RepositoryActions({ repository, mutateList, onVisit }: R
           ))}
         </ActionPanel.Submenu>
 
+        <Action
+          icon={Icon.Terminal}
+          title="Clone and Open"
+          onAction={() => cloneAndOpen(repository)}
+          shortcut={{ modifiers: ["cmd", "opt"], key: "c" }}
+        />
+
         <Action.OpenInBrowser
           icon="vscode.svg"
           title="Clone in VSCode"
```

Comment:
I know there was already this command available. However, I think it's very limited and nowhere near as powerful as the new one.
It allows you to open VS Code with the clone prompt but doesn't let you open an already-cloned repository from the search results. That's why I added the Clone and Open: you clone once and open as many times as you want.
In my opinion, we could deprecate the Clone in VSCode command. However, it would be a breaking change, and I don't know what's the best practice in this case. Personally, I don't have any preference in this regard, and we could leave it there.

Reply:
I think it's fine having them both. The VS Code action always prompt you to select the repository destination so it's slightly different.